src/data/__tests__/dataOperationsProvider.test.ts

```diff
@@ -1,7 +1,7 @@
-import { VectorOperationsProvider } from '../vectorOperationsProvider';
+import { DataOperationsProvider } from '../dataOperationsProvider';
```

Comment:
See `dataOperationsProvider` file below, renamed to align with the new API class.